k8/base/cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: hello
```

Comment:
Please pick a name that identifies the purpose of the cron job (otherwise someone will see resource named "hello" and wonder what is going on).